docs/api/app.md

```diff
@@ -604,6 +604,8 @@ You can request the following paths by the name:
   * `~/Library/Application Support` on macOS
 * `userData` The directory for storing your app's configuration files, which by
   default it is the `appData` directory appended with your app's name.
+* `appCache` The directory for storing your app's Electron cache, which by
+  default it is the `appData` directory appended with your app's name.
```

Comment:
Document should warn user that setting `appCache` would make app incompatible with old data and they need to migrate data from old dir to new dir themselves.

Reply:
Using `appCache` as key is quite confusing as `app`-prefix has other meanings in `appData`. Maybe `browserCache` is better.